shared/js/background/chrome-events.es6.js

```diff
@@ -167,11 +167,14 @@ const redirect = require('./redirect.es6')
 const tabManager = require('./tab-manager.es6')
 const pixel = require('./pixel.es6')
 const https = require('./https.es6')
-const cookieConfig = require('./../background/storage/cookies.es6')
 
 const requestListenerTypes = utils.getUpdatedRequestListenerTypes()
 
 function blockTrackingCookies () {
+    if (tdsStorage.config.features) {
```

Comment:
There was no way to disable third party cookies before?

Reply:
Not globally only per domain.